ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -802,14 +796,17 @@ public function db_connect( $query = '' ) {
 					continue;
 				}
 
-				$this->run_callbacks( 'db_connection_error', array(
-					'host'      => $host,
-					'port'      => $port,
-					'operation' => $operation,
-					'table'     => $this->table,
-					'dataset'   => $dataset,
-					'dbhname'   => $dbhname
-				) );
+				$this->run_callbacks(
```

Comment:
Generally, I do not like the extra indentation, but this is the kind of change that does not seem to make anything worse, so it is probably OK.